theano/sandbox/cuda/basic_ops.py

```diff
@@ -3041,24 +3041,6 @@ def make_node(self, *axis_and_tensors):
         as_tensor_variable_args = [as_cuda_ndarray_variable(x)
                                    for x in tensors]
 
-        # Get joining axis as int
```

Comment:
You want constant axis to be set to 0 in make_node while symbolic axis can get to {perform,c_code} as a negative value?